linux/just_files/new_just

```diff
@@ -469,118 +479,120 @@ function write_dockerfile()
 {
   exists "${1}" && return 0
 
-  uwecho   'FROM vsiri/recipe:gosu as gosu
+  uwecho   '#T# Dockers use gosu instead of sudo because sudo is poorly designed for use in a docker
+  uwecho   '#T# Note: A function "sudo" is added to call "gosu root" for convenience
```

Comment:
```suggestion
  uwecho   '#T# Dockers use gosu instead of sudo because sudo is a poor fit for docker.
  uwecho   '#T# Note: A function "sudo" is added to call "gosu root" for convenience
```